hosted_control_planes/hcp-backup-restore-dr.adoc

```diff
@@ -11,10 +11,20 @@ If you need to back up and restore etcd on a hosted cluster or provide disaster
 :FeatureName: Hosted control planes
 include::snippets/technology-preview.adoc[]
 
+[id="hosted-etcd-non-disruptive-recovery"]
+== Recovering etcd pods for hosted clusters
+
+In hosted clusters, etcd pods run as part of a stateful set. The stateful set relies on persistent storage to store etcd data for each member. In a highly available control plane, the size of the stateful set is 3, and each member has its own persistent volume claim.
```

Comment:
Per ISG, spell out numbers zero through nine.
Do we need to say three what? Pods? Members? (Are these different?)

Reply:
Good catch! I'll update the text to clarify that it's three members.